mathics/doc/common_doc.py

```diff
@@ -1058,6 +1055,8 @@ def __init__(self, module=None):
                 and var.__module__[: len(self.pymathicsmodule.__name__)]
                 == self.pymathicsmodule.__name__
             ):  # nopep8
+                if not check_requires_list(var):
+                    continue
```

Comment:
I am not seeing that the "continue" ever gets used.

Reply:
This block is actually never used, because we didnt implement the integrated pymathics documentation

Reply:
Ok. Then either this should be noted in the code or removed. 

Personally, I hope we redo all of this using Python's doctest instead of going further down this mess with "integrated pymathics documentation".

Reply:
Yes, that was a reason for which I didn't continue working in that direction.

Reply:
Let's remove the pymathics stuff then.

Reply:
I tried to do that, but for some reason 3/4 of the pytests disappeared. I need to investigate it a little bit more.

Reply:
@mmatera I am currently seeing that in the this PR right now. One place this went off is between  [4905ab3](https://github.com/Mathics3/mathics-core/pull/376/commits/4905ab3869585547db2ba9ebb1326e4803abba2a) and 6e74bb78bd44ae92301f61b5e15bb8112373689a where I removed what looks like an undefined `package` variable but that seems to have a magical effect _when packages are installed_ but throws an error when the packages aren't installed. See [this window build](https://github.com/Mathics3/mathics-core/runs/6955914974?check_suite_focus=true)

This PR is now to big and unwieldy. Following the practice I mentioned before I am going to start splitting this up into smaller separable pieces that can go in.

Reply:
Let me do that in my repo. We can leave it for a while, until you finish with the avoiding conversion stuff.

Reply:
Ok for the "check_requires" part. I have 3 PRs for subsidiary things that that are here and should be split off.
(Actually, I think there may be one more to catch some outright small typos). 

Let's get these easy things in. The PyPy stuff does help in allowing benchmarking PyPy vs pyston.